components/RouteInfoModal.tsx

```
@@ -0,0 +1,129 @@
+                </button>
+              </div>
+            </ModalHeader>
+
+            <ModalBody>
+              <p className="font-bold">停車駅: </p>
+              <div className="flex flex-wrap gap-x-2">
+                {dropEitherJunctionStation(route?.stops ?? []).flatMap((stop) =>
+                  stop.stopCondition === StopCondition.All ? (
+                    <span>{stop.name}</span>
+                  ) : (
+                    <span
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)